superset-frontend/src/components/Card/index.tsx

```diff
@@ -28,11 +28,15 @@ const Card = ({ padded, ...props }: CardProps) => (
   <AntdCard
     {...props}
     css={(theme: SupersetTheme) => ({
+      // 'border-radius': `${theme.gridUnit}px`,
+      border: `1px solid ${theme.colors.grayscale.light2}`,
```

Comment:
@geido I think both of these lines of styling can be replaced by AntD "tokens" but this where my problem lies.... I can enable the border-radius here and make it `gridUnit * 10` and I see the difference. if I make the `token` be `borderRadius * 10` it seems to make no difference in the UI.